src/decorators/dispatch.ts

```diff
@@ -17,9 +17,11 @@ export function dispatch(): PropertyDecorator {
 
     const wrapped = function (this: any, ...args: any[]) {
       const result = originalMethod.apply(this, args);
-      const store = getBaseStore(this) || NgRedux.instance;
-      if (store) {
-        store.dispatch(result);
+      if (result !== NgRedux.DISPATCH_NOOP) {
```

Comment:
As discussed in the PR - I think having this be checking for undefined, or a falsy value instead of a special action-type/magic string would be a better fit. 

Leaving it at the decorator level like this is appropriate and I like it (instead of trying to change the actual store.dispatch), as these decorators/helpers are more geared towards smoothing out use with Angular and going a step beyond providing a binding between Angular and Redux.

I recently updated master tweaking some of the style / formatting rules to use prettier - so you may need to rebase and run the formatter also.